modules/cluster-logging-collector-log-forward-project.adoc

```diff
@@ -66,20 +67,23 @@ spec:
 <5> Specify the URL and port of the external log aggregator as a valid absolute URL. If the cluster-wide proxy using the CIDR annotation is enabled, the output must be a server name or FQDN, not an IP address.
 <6> If using a `tls` prefix, you must specify the name of the secret required by the endpoint for TLS communication. The secret must exist in the `openshift-logging` project and have *tls.crt*, *tls.key*, and *ca-bundle.crt* keys that each point to the certificates they represent.
 <7> Configuration for an input to filter application logs from the specified projects.
-<8> Configuration for a pipeline to use the input to send project application logs to an external Fluentd instance.
-<9> The `my-app-logs` input.
-<10> The name of the output to use.
-<11> Optional: String. One or more labels to add to the logs.
-<12> Configuration for a pipeline to send logs to other log aggregators.
-** Optional: Specify a name for the pipeline.
-** Specify which log types to forward by using the pipeline: `application,` `infrastructure`, or `audit`.
-** Specify the name of the output to use when forwarding logs with this pipeline.
-** Optional: Specify the `default` output to forward logs to the internal Elasticsearch instance.
-** Optional: String. One or more labels to add to the logs.
+<8> If no namespace is specified, logs are collected from all namespaces.
+<9> Configuration for a pipeline to use the input to send project application logs to an external Fluentd instance.
+<10> The `my-app-logs` input.
+<11> The name of the output to use.
+<12> Optional: String. One or more labels to add to the logs.
+<13> Configuration for a pipeline to send logs to other log aggregators.
++
+* Optional: Specify a name for the pipeline.
+* Specify which log types to forward by using the pipeline: `application,` `infrastructure`, or `audit`.
+* Specify the name of the output to use when forwarding logs with this pipeline.
+* Optional: Specify the `default` output to forward logs to the internal Elasticsearch instance.
+* Optional: String. One or more labels to add to the logs.
+<14> Note that application logs from all namespaces are collected when using this configuration.
```

Comment:
Some of my nits and the extra explanation here might be simplified by a link to the reference docs. Those give a detailed definition for each field independently. Then this can be treated more like an example, with less pressure to explain every detail.

Reply:
We can't put xref links in modules